mods/cnc/rules/map-generators.yaml

```diff
@@ -330,18 +330,8 @@
 						MaximumBuildings: 3
 						BuildingWeights:
 							hosp: 2
-							miss: 1
+							bio: 1
```

Comment:
Replacing `miss` with `bio` wasn't in the PR description. Is it intended?

I think adding `bio` for CnC is reasonable (it's not as disruptive as RA's), but I don't see a reason to remove `miss`?